ttrss-config.php

```diff
@@ -89,7 +89,11 @@
 	// *** Feed settings ***
 	// *********************
 
-	define('FORCE_ARTICLE_PURGE', 5);
+        $force_article_purge = 5;
+        if (getenv('FORCE_ARTICLE_PURGE') !== null) {
+            $force_article_purge = getenv('FORCE_ARTICLE_PURGE');
```

Comment:
maybe it is a good idea to force the conversion to int? (by casting with `(int)` or by calling `intval()`)